### PR TITLE
Add optionnal standard numbering anime search

### PR DESCRIPTION
**Pull request by Bakeneko**
#### Database Migration
NO

#### Description
Add an indexer setting to enable additionnal standard numbering format searches for anime.

#### Todos
- [x] Tests

#### Issues Fixed or Closed by this PR
Related to #4153

### CODE REVIEW

File: src/NzbDrone.Core/Indexers/Fanzub/FanzubSettings.cs [hunk 1]
```diff
@@ -25,6 +25,9 @@ public FanzubSettings()
         [FieldDefinition(0, Label = "Rss URL", HelpText = "Enter to URL to an Fanzub compatible RSS feed")]
         public string BaseUrl { get; set; }
 
+        [FieldDefinition(1, Label = "Anime seasonal search", Type = FieldType.Checkbox, HelpText = "Add seasonal format search for Anime")]
```
Comment (markus101) [inline src/NzbDrone.Core/Indexers/Fanzub/FanzubSettings.cs:28]: This format is referred to as `Standard` everywhere else in Sonarr, it should be referred to that way in the setting as well, Season/Seasonal is far too easily confused with season pack searches.

Maybe: `Anime Standard Format Search` with the help text of `Also search for anime using the standard numbering format`

**Review by markus101 — CHANGES_REQUESTED** [review]
Made a suggestion to improve the naming, but season anime searches should also search for standard season packs if that option is enabled.

**Comment by Bakeneko** [comment]
Added @markus101 suggestions and tests for request generators.

**Comment by markus101** [comment]
Made a small change and landed this in cee17483d97c1414b05b9f19729d612ef43934c2

**Comment by reloxx13** [comment]
Heya,

when this checkbox is checked, shouldn't it also use t=TVSEARCH and not t=SEARCH for animes?

Standard uses tvsearch, while anime uses search, even when `Anime Standard Format Search` is checked.
When using t=SEARCH with anime as type, the results are ignoring the searched season and episode and returning alot of unwanted/unasked episodes. while changing t=TVSEARCH (i did it manual with the previos SEARCH api call) is returning the correct episodes.

if i set the anime serie to standard and add the anime category to the search categories of standard, im getting the correct episodes returned.

**Pull request closed without merging**